src/Service.GraphQLBuilder/GraphQLUtils.cs

```diff
@@ -248,6 +248,7 @@ public static ObjectType UnderlyingGraphQLEntityType(IType type)
                     INT_TYPE => new(INT_TYPE, new IntValueNode(int.Parse(defaultValueFromConfig))),
                     LONG_TYPE => new(LONG_TYPE, new IntValueNode(long.Parse(defaultValueFromConfig))),
                     STRING_TYPE => new(STRING_TYPE, new StringValueNode(defaultValueFromConfig)),
+                    UUID_TYPE => new(UUID_TYPE, new UuidType().ParseValue(Guid.Parse(defaultValueFromConfig))),
```

Comment:
Are there any other expected **supported** types that might be missing handling in this conversion? We should at least identify what else is missing.. 

Its could be a different issue to add new unsupported types but need to ensure all types that we claim to support are supported everywhere - be it stored proc parameter or view column type